### PR TITLE
KK-215 | Add usage analytics

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/common/components/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -42,13 +43,16 @@ const Dropdown: React.FunctionComponent<DropdownProps> = ({
 
   const ariaLabel = isOpen
     ? t('common.menu.closeMenuText')
+    : options.length === 1
```
Comment (midori0507) [inline src/common/components/dropdown/Dropdown.tsx]: where is this idea come from?

Comment (midori0507) [reply src/common/components/dropdown/Dropdown.tsx]: The double chaining is really bad for reading just saying

Comment (badrange) [reply src/common/components/dropdown/Dropdown.tsx]: I know it is bad, I was too lazy to make anything better right now, maybe we have too little oxygen in the office. I will come up with something better tomorrow unless you have a suggestion out of hand. 

The problem is that the aria-label & aria-expanded info is totally wrong on the user/drop down menu on desktop view. It always stays as `Open menu` even if it should be `Log in`.

Comment (midori0507) [reply src/common/components/dropdown/Dropdown.tsx]: I can commit straight to make it more readable, take some rest.

Comment (badrange) [reply src/common/components/dropdown/Dropdown.tsx]: Be my guest! 😻

**Pull request merged**